### PR TITLE
aws-env: init at 0.4

### PR DESCRIPTION
**Pull request by ento**
###### Motivation for this change

This PR adds aws-env, a useful tool for pulling environment-specific variables from AWS Systems Manager Parameter Store.

The contributing guide requires that `meta.maintainers` be set; I'm not really good at being a maintainer - what'd be the protocol in that case?

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by srhb** [comment]
@GrahamcOfBorg build aws-env

@ento I don't believe it's a hard requirement that new packages absolutely *must* have a maintainer, but it certainly is preferable by a large margin, so that people at least have someone to ping on github to ask for reviews of version bumps etc.

**Review by srhb — CHANGES_REQUESTED** [review]

File: pkgs/tools/admin/aws-env/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{ stdenv, buildGoPackage, fetchgit, lib }:
+
+buildGoPackage rec {
+  name = "aws-env-${version}";
+  version = "0.4";
+  rev = "82c83402547c73a63bc478af08b156da1ba363e6";
+
+  goPackagePath = "github.com/Droplr/aws-env";
+
+  src = fetchgit {
```
[the remark this answers is not in the document]
Comment (ento) [reply pkgs/tools/admin/aws-env/default.nix]: Done: 0e1e742

**Comment by ento** [comment]
D'oh, added to all-packages.nix in 6e8a5b0. I can squash the commits if that's preferable.

re: maintainer - got it. Is the ping more for Nix expertise or for knowledge of the package content itself?

**Comment by srhb** [comment]
@ento I'd say in this case where it's a relative simple package, the question that people will mostly be aiming at the maintainer is probably "hey, I updated this package, is that OK?"

And yes please, squash 'em. :)

**Comment by srhb** [comment]
@GrahamcOfBorg build aws-env

**Comment by ento** [comment]
They've been squashed.

> "hey, I updated this package, is that OK?"

Being a mere user of aws-env, I'll only be able to say "I guess?" and double-check any open issues for aws-env at most. So I'll cc @orfin for any interest in having your name listed as a maintainer / point of contact for aws-env in [Nix](https://nixos.org/nix/) (if there's interest, we can update the package metadata in a separate PR), and ask the good reviewer(s) of nixpkgs to consider it a non-blocker to have no maintainer listed for this package for now.

**Comment by srhb** [comment]
@ento Alright, but I'd feel bad adding a package with no maintainer, so can you please add me (`stdenv.lib.maintainers.srhb`) for now and resquash, and I'll take it upon me to forward any request that requires deeper knowledge of the tool to relevant people.

**Review by srhb — APPROVED** [review]

**Comment by ento** [comment]
@srhb Thank you, much appreciated. I've updated `maintainers` and force-pushed.

**Comment by srhb** [comment]
Thank you. :)

**Pull request merged**